superset-frontend/src/dashboard/components/nativeFilters/FilterBar/FilterBar.test.tsx

```diff
@@ -57,8 +53,25 @@ class MainPreset extends Preset {
   }
 }
 
-fetchMock.get(`glob:*/api/v1/dataset/${datasourceId}`, {
-  result: [mockDatasource[fullDatasourceId]],
+fetchMock.get(`glob:*/api/v1/dataset/1`, {
```

Comment:
Curious why we need to move away from the `mockDatasource` - whether it's for simplicity, or if there's something that indicates an incompatibility between the mock data and the test itself.

Reply:
Followup question - do we need to update the `mockDatasource` in any way so that it's _not_ incompatible? Just don't want to carry any tech debt forward.

Reply:
I can try to update in the next iteration and see if it raises other problems.